libraries/templates/compare_stats.html

```diff
@@ -23,7 +23,7 @@ <h1 style="padding-bottom:35px">Compare Library Stats</h1>
         <tr id="username">
           <th scope="col"></th>
           <th scope="col" class="field">{{ my_library.user.username }}</th>
-          <th scope="col" class="field>Difference</th>
+          <th scope="col" class="field">Difference</th>
```

Comment:
Someone did a bad thing. We can talk about how to mitigate something like this in the future.